guillotina/traversal.py

```diff
@@ -226,7 +226,24 @@ def debug(self, request, resp):
     cors_renderer = app_settings['cors_renderer'](request)
     try:
         cors_headers = await cors_renderer.get_headers()
-        cors_headers.update(resp.headers)
+        fields = (
+            'Access-Control-Expose-Headers',
+            'Access-Control-Allow-Methods',
+            'Access-Control-Allow-Headers'
+        )
+        for name, value in resp.headers.items():
+            if name in fields and name in cors_headers:
+                if value == '*':
+                    cors_headers[name] = '*'
+                elif cors_headers[name] != '*':
+                    cors_values = cors_headers[name].split(', ')
```

Comment:
Maybe to be safe we should split on `,` instead of `, `?

`cors_values = [v.strip() for v in cors_headers[name].split(',)]`

Reply:
right :+1: